typescript/cli/src/config/chain.ts

```diff
@@ -90,6 +90,62 @@ export async function createChainConfig({
     protocol: ProtocolType.Ethereum,
     rpcUrls: [{ http: rpcUrl }],
   };
+  const wantAdvancedConfig = await confirm({
+    message: 'Do you want to add advanced config for this chain?',
```

Comment:
Let's be more specific about what's being offered here. I suggest:
`Do you want to set block or gas properties for this chain config (optional)`

Reply:
Sure i will update it !!!